product_main_supplier/views/product_template_views.xml

```diff
@@ -12,4 +12,48 @@
         </field>
     </record>
 
+    <!-- Should this view inherit and modify rather than replace? -->
+    <record model="ir.ui.view" id="product_template_tree_view">
+        <field name="name">product.template.main.seller.tree</field>
+        <field name="model">product.template</field>
+        <field name="priority">14</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+                <field name="default_code" />
+                <field name="categ_id" />
+                <field name="main_seller_id" string="Main Seller" />
+                <field
+                    name="main_seller_id_product_code"
+                    string="Main Seller Product Code"
+                />
+                <field name="weight" />
+                <field name="main_price" />
+                <field name="main_minimum_qty" />
+                <field name="uom_po_id" />
+                <field name="qty_available" />
+                <field name="virtual_available" />
+                <field name="stock_coverage" />
+                <field name="daily_sales" />
+                <field name="range_sales" />
+                <field name="list_price" />
+                <field name="effective_sale_price" />
+            </tree>
+        </field>
+    </record>
+
+    <!-- These look like customizations rather than functionalities,
```

Comment:
This can stay in `beesdoo_product_usability`

Reply:
done